src/query-service/swagger/query-service.yaml

```
@@ -1197,6 +1197,19 @@ components:
         type: integer
         format: unsigned
         description: The number of warnings that occurred during the request.
+      sessionMemory:
+        type: integer
+        format: bytes
+        description: >
+          The memory session size for the request, in bytes.
+
+          
+          Each request has a dedicated memory session. 
+          When a query requires a document or value, memory is allocated from this session based on the size of the document or value.
+          Once the document or value is processed, the allocated memory is returned back to the session, enabling it to be reused by the request. 
+
+          
+          This metric is available only when `node quota` and `node-quota-val-percent` are configured for the node.
```

Are you going to link to `node-quota` and `node-quota-val-percent` on the Query Admin REST API page?

(Even if not, `node quota` needs a hyphen)

```suggestion
          This metric is available only when `node-quota` and `node-quota-val-percent` are configured for the node.
```